warp-tls/Network/Wai/Handler/WarpTLS.hs

```diff
@@ -18,8 +18,8 @@ module Network.Wai.Handler.WarpTLS (
     , tlsWantClientCert
     , tlsServerHooks
     , defaultTlsSettings
-    , tlsSettings
```

Comment:
Does this have to be a breaking change? That makes me very hesitant to move ahead with it.